frontend/src/utils/date-utils.ts

```diff
@@ -7,6 +7,7 @@ export interface StartEndDate {
 }
 
 const millisecondsInADay = 24 * 60 * 60 * 1000;
+const millisecondsInAYear = 365 * millisecondsInADay;
```

Comment:
This could be slightly off due to bissextile years. Could this be an issue?

Reply:
Yea, I think this could create issues (specifically, season bounds being a day off). Nice catch, I'm going to refactor to prevent this.

Reply:
Updated in 07afa32